core/platform/models.py

```diff
@@ -492,3 +506,12 @@ def import_search_services(cls):
             module. The search_services module.
         """
         return cls._get().import_search_services()
+
+    @classmethod
+    def import_storage_services(cls):
```

Comment:
This seems wrong? (In terms of docstring)

Reply:
Done.